woocommerce/woocommerce-gutenberg-products-block
# Split blocks into separate JS files

This lets us use the functionality of `register_block_type` to automatically enqueue the scripts and styles used by blocks in the editor, and eventually, styles on the frontend as well. Also adds cssnano to minify our CSS.

Why go back to separate files after combining them in #220 ? By having each block as its own file, we can take advantage of the `register_block_type` php function to automatically enqueue our CSS on the frontend, which will be useful for the Featured Product block, and also when we move to the custom grid display.

There should be no functionality changes.

### How to test the changes in this Pull Request:

1. Load each block, check that they still work as expected.

Note: I need to figure out a solution for separating out editor & front end styles via webpack, but this is eluding me at the moment. This will work for now, so I'm switching to "needs review".
